Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -221,7 +167,7 @@ private static void OnWindowFocusChange(object sender, WindowChangedEventArgs ev
             WindowFocusChange?.Invoke(sender, eventArgs);
         } 
 
-        private static ICodePane GetCodePaneFromHwnd(IntPtr hwnd)
+        public static ICodePane GetCodePaneFromHwnd(IntPtr hwnd)
```

Comment:
Why can't this be a method or a ctor on the `ICodePane`?

Reply:
The issue there is that in order to get subclasses established as soon as possible, we're attaching them *way* before the VBE objects are ready to handle requests. That's also the reason why the `CodePaneSubclass` was changed to allow lazy association with its `ICodePane`.